README.md

```diff
@@ -2,6 +2,44 @@
 
 Pre-release development of high-level probabilistic programming interface for TensorFlow.  Please contribute or participate [on github](https://github.com/pymc-devs/pymc4).
 
+## Installation Instructions
+ - Install using `pip`
+ ``` bash
+ pip install git+https://github.com/pymc-devs/pymc4.git
+ ```
+ 
+## Simple Example
+### Import pymc4 and Edward2
+``` python
+import pymc4 as pm
+from tensorflow_probability import edward2 as ed
+```
+### Model Initialization
+``` python
+model = pm.Model()
+```
+
+### Model Definition
+The model has to be defined in a single function with `@[model-name].define` decorator.
+``` python
+@model.define
+def simple(cfg):
+    normal = ed.Normal(loc=0. scale=1., name='normal')
+```
+
+### Sampling
+``` python
+trace = pm.sample(model)
+```
+
+### Visualize the trace using arviz
+``` python
+# Add code here
```

Comment:
@ColCarroll You mentioned being able to use arviz with pymc4 trace dictionary output last week. Could you please let me know the exact syntax?

Reply:
```python
# See https://github.com/arviz-devs/arviz
# pip install git+git://github.com/arviz-devs/arviz.git
import arviz as az

posterior_data = az.convert_to_xarray(trace, chains=1)
az.posteriorplot(posterior_data, figsize=(8, 4), textsize=15, round_to=2);
```

![image](https://user-images.githubusercontent.com/2295568/43688039-c1bd0d28-98e1-11e8-92fb-bf9652259900.png)